src/igvfd/schemas/construct_library_set.json

```diff
@@ -132,59 +138,161 @@
+                                            {
+                                                "const": "exon"
+                                            },
+                                            {
+                                                "const": "genes"
+                                            }
+                                        ]
+                                    }
                                 },
-                                {
-                                    "required": [
-                                        "lower_bound_guide_coverage"
+                                "not": {
+                                    "anyOf": [
+                                        {
+                                            "required": [
+                                                "guide_type"
+                                            ]
+                                        },
+                                        {
+                                            "required": [
+                                                "tiling_modality"
+                                            ]
+                                        },
+                                        {
+                                            "required": [
+                                                "average_guide_coverage"
+                                            ]
+                                        },
+                                        {
+                                            "required": [
+                                                "lower_bound_guide_coverage"
+                                            ]
+                                        },
+                                        {
+                                            "required": [
+                                                "upper_bound_guide_coverage"
+                                            ]
+                                        }
                                     ]
+                                }
+                            },
+                            {
+                                "required": [
+                                    "large_scale_gene_list"
+                                ],
+                                "properties": {
+                                    "scope": {
+                                        "oneOf": [
+                                            {
+                                                "const": "tile"
+                                            },
+                                            {
+                                                "const": "exon"
+                                            },
+                                            {
+                                                "const": "genes"
+                                            }
+                                        ]
+                                    }
```

Comment:
i dont think large-scale gene list could be exon/tile?

Reply:
fixed